.ai/auto-review/env_vars.py

```diff
@@ -30,8 +30,7 @@ def __init__(self):
             "token" : self.token,
             "base_ref" : self.base_ref,
             "pull_number" : self.pull_number,
-            "chat_gpt_token" : self.chat_gpt_token,
-            "chat_gpt_model" : self.chat_gpt_model,
+            "gemini_ai_token" : self.gemini_ai_token
         }
 
```

Comment:
35: AttributeError 'EnvVars' object has no attribute 'target_extensions' Accessing an undefined attribute after its potential initialization. It seems like `self.target_extensions` is intended to be initialized based on the environment variable `TARGET_EXTENSIONS`. Ensure this variable is set and handled correctly.